### PR TITLE
Added New XIAO_WS_RP2040_Blink_Common Example Project

### PR DESCRIPTION
**Pull request by ab-tools**
This adds a new example project that combines the `Blink` one with `OGM-Common` as KNX back-end.

That was missing for me while getting started, so might be useful for somebody else as well.

I've included build configurations for Seeed XIAO RP2040 and WaveShare RP2040-Zero.

### CODE REVIEW

File: XIAO_WS_RP2040_Blink_Common/extra_script_seeed_xiao_rp2040.py
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/extra_script_seeed_xiao_rp2040.py:1]: Not needed anymore

File: XIAO_WS_RP2040_Blink_Common/extra_script_waveshare_rp2040_zero.py
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/extra_script_waveshare_rp2040_zero.py:1]: Not needed anymore

File: XIAO_WS_RP2040_Blink_Common/include/versions.h
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/include/versions.h:1]: This will be generated automaticly

**Comment by thewhobox** [comment]
thanks for your contribution.

I pushed a new commit with all needed changes to get it work.
Please also make this changes in your example and check if it still compiles.

Then i can merge your example.

File: XIAO_WS_RP2040_Blink_Common/lib/README
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/lib/README:1]: Remove this folder, otherwise the scripts will fail.

File: XIAO_WS_RP2040_Blink_Common/platformio.ini
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/platformio.ini:1]: See this changes: [PiPico_RP2040_Common/platformio.ini](https://github.com/OpenKNX/Examples/commit/5086a143988f33128f2500eb414bb6fa2e847877#diff-80f40cf9c3bd1612a9f87018d8915274af1c71533c4178bb568af99bd31037f7)

File: XIAO_WS_RP2040_Blink_Common/src/main.cpp
Comment (thewhobox) [inline XIAO_WS_RP2040_Blink_Common/src/main.cpp:1]: You can get rid of lines 7-9
It is not neccessary to run.

**Comment by ab-tools** [comment]
Thanks, Mike, all done and compiles fine for me!

Please note that I did not add the `khoih-prog/TimerInterrupt_Generic` library dependency back in as it seems to add this automatically as sub dependency. So I guess we don't need to link this explicitly, do you agree?

**Comment by thewhobox** [comment]
Yes, the  khoih-prog/TimerInterrupt_Generic is used by Common and is defined in the library.json

Thank you for your changes

**Pull request merged**